release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -193,6 +193,24 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Bare Metal Hardware Provisioning*
+
+* Previously, Ironic failed to download an image for installation because it uses HTTPS by default and Ironic does not have the correct certificate bundle available. This issue is fixed by setting the image download as insecure to request a transfer without the certificate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953795[*BZ#1953795*])
+
+* Previously, when using dual stack deployments, worker node hostnames sometimes did not match the name inspected before deployment. This caused nodes to need manual approval. This has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955114[*BZ#1955114*])
+
+* Previously, in UEFI mode, the ironic-python-agent created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the ironic-python-agent configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966129[*BZ#1966129*])
+
+* Previously, a node sometimes selected the incorrect IP version upon startup (v6 instead of v4, or vice versa). The node would fail to come up because it did not receive an IP address. This is fixed by cluster-baremetal-operator passing the IP Option to the downloader (ip=dhcp or ip=dhcp6), so this is set correctly at startup and the node comes up as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946079[*BZ#1946079*])
+
+*Assisted Installer*
+
+* Previously, assisted-service did not wait for postgres to start up and be ready to accept connections. Assisted-service attempted to establish a database connection, failed, and assisted-service container failed and restarted. This issue has been fixed by assisted-service attempting to connect to the database for up to 10 seconds. If postgres starts and is ready to accept connection within 10 seconds, assisted-service will connect without going into an error state. If assisted-service is unable to connect to postgres within 10 seconds, it will go into an error state, restart, and try again. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1941859[*BZ#1941859*])
+
+*RHCOS*
+
+* Previously, the SELinux policy was not allowing systemd to read files under /etc/kubernetes, and the kubelet failed to start. This is fixed by updating the SELinux policy to allow systemd to read files with kubernetes_file_t labels. The kubelet starts successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969998[*BZ#1969998*])
```

Comment:
The systemd process has some kind of immunity from being formatted as a daemon--I have no idea why.  Please format `/etc/kubernetes` in backticks because it is a directory and formatting directories is covered in the [quick reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) section of the guidelines.

Kubelet also seems to have the same immunity from formatting that systemd has.

If `kubernetes_file_t` is a system term or item as identified in the quick reference, then please enclose it in backticks.